driver/pelican/interface.md

```diff
@@ -6,24 +6,26 @@ The following structs define the way users interact with the Scheduling interfac
 
 ##### Schedule Structs
 
-// Struct mapping each day of the week to its daily schedule <br>
-type ThermostatSchedule struct {<br>
-&nbsp;&nbsp;&nbsp;DaySchedules map[string]ThermostatDaySchedule `msgpack:"day_schedules"`<br>
-}<br>
-
-// Struct containing a series of blocks that describes a one day schedule <br>
-type ThermostatDaySchedule struct { <br>
-&nbsp;&nbsp;&nbsp;Blocks []ThermostatBlockSchedule `msgpack:blocks` <br>
-} <br>
-
-// Struct containing data defining the settings of each schedule block <br>
-type ThermostatBlockSchedule struct { <br>
-&nbsp;&nbsp;&nbsp;CoolSetting float64 `msgpack:"cool_setting"` <br>
-&nbsp;&nbsp;&nbsp;HeatSetting float64 `msgpack:"heat_setting"` <br>
-&nbsp;&nbsp;&nbsp;System      string  `msgpack:"system"` <br>
-&nbsp;&nbsp;&nbsp;Time        string  `msgpack:"time"` <br>
+```
+// Struct mapping each day of the week to its daily schedule
+type ThermostatSchedule struct {
+  DaySchedules map[string]ThermostatDaySchedule `msgpack:"day_schedules"`
 }
 
+// Struct containing a series of blocks that describes a one day schedule
+type ThermostatDaySchedule struct {
+  Blocks []ThermostatBlockSchedule `msgpack:blocks`
```

Comment:
Can you put quotes around `blocks` so its consistent with the other annotations?

Reply:
Got it! I'll be more consistent going forwards.